bff/src/Bff/EndpointServices/Login/DefaultLoginService.cs

```diff
@@ -13,10 +14,19 @@ namespace Duende.Bff;
 /// </summary>
 public class DefaultLoginService : ILoginService
 {
+    /// <summary>
+    /// Authentication scheme provider
+    /// </summary>
+    protected readonly IAuthenticationSchemeProvider AuthenticationSchemeProvider;
```

Comment:
I think these should be properties and not fields?

Reply:
I think they shouldn't be there at all. Will remove these in a future version.